mm2src/coins/nft/nft_errors.rs

```diff
@@ -49,6 +74,21 @@ impl From<GetEthAddressError> for GetNftInfoError {
     fn from(e: GetEthAddressError) -> Self { GetNftInfoError::GetEthAddressError(e) }
 }
 
+impl From<CreateNftStorageError> for GetNftInfoError {
+    fn from(e: CreateNftStorageError) -> Self {
+        match e {
+            CreateNftStorageError::Internal(err) => GetNftInfoError::Internal(err),
+        }
+    }
+}
+
+impl<T: NftStorageError> From<T> for GetNftInfoError {
+    fn from(err: T) -> Self {
+        let msg = format!("{:?}", err);
+        GetNftInfoError::DbError(msg)
```

Comment:
I think this can be used straight away no need to declare extra var
```rust 
GetNftInfoError::DbError(format!("{:?}", err))
```

Reply:
thanks, done